### ISSUE TITLE
OIDCPassRefreshToken is set to true instead of On in /opt/rh/httpd24/root/etc/httpd/conf.d/ood-portal.conf

### ISSUE THREAD

**Issue by xpillons** [OSC/ood-ansible#116]
When defining OIDCPassRefreshToken like below on an overrides.yml file
```yml
oidc_settings:
  OIDCPassIDTokenAs: 'serialized'
  OIDCPassRefreshToken: 'On'
  OIDCPassClaimsAs: 'environment'
```
Then after the update generator has been run the OIDCPassRefreshToken is set to true instead of On

```
[root@ondemand hpcadmin]# grep OIDCPassRefreshToken /opt/rh/httpd24/root/etc/httpd/conf.d/ood-portal.conf
  OIDCPassRefreshToken true
```

And below is the error message when starting `httpd24-httpd.service`
```
systemctl status httpd24-httpd.service -l

Jul 06 08:06:41 ondemand httpd-scl-wrapper[10636]: AH00526: Syntax error on line 95 of /opt/rh/httpd24/root/etc/httpd/conf.d/ood-portal.conf:
Jul 06 08:06:41 ondemand httpd-scl-wrapper[10636]: OIDCPassRefreshToken must be On or Off
```

This is using version 1.8

**Comment by johrstrom** [comment]
In #115 you'd added spaces, you should also add quotes for jinja/python to treat them as strings and not booleans. 

I just removed the `| quote` filter in #117 and just quoted everything directly.

**Comment by johrstrom** [comment]
This was fixed in #117 and released in 2.0.2.  If anyone needs this back ported to 1.8, just comment on the ticket.
